lib/openshift/flakes/conditions.rb

```diff
@@ -0,0 +1,40 @@
+module BushSlicer
+
+  # this class should help with parsing report conditions
+  class Conditions
+
+    attr_reader :struct
+    private :struct
+
+    def initialize(struct)
+      @struct = struct
+    end
+
+    module ExportMethods
```

Comment:
I don't think we need this pattern for classes. For modules that we want to include or use methods from them directly it makes sense. But I don't see why we need to have `ExportMethods` module inside the class.